packages/devtools_app/test/flutter/logging_screen_test.dart

```diff
@@ -142,7 +144,9 @@ void main() {
         await tester.tap(find.byKey(ValueKey(fakeLogData[996])));
         await tester.pumpAndSettle();
         expect(
-          find.text('log event 996'),
+          find.byWidgetPredicate((widget) =>
```

Comment:
there is a test helper for finding a widget with matching text directly rather than having to match a predicate.

Reply:
It looks like `find.text` only works for `Text` and `EditableText` widgets. Ian suggested either adding keys to rich text or using this type of predicate (https://stackoverflow.com/questions/41152320/is-there-an-easy-way-to-find-particular-text-built-from-richtext-in-a-flutter-te). We could consider keys too, although maybe it's better to use a predicate to avoid adding code that's just for testing.

Reply:
That is too bad. We will be doing enough rich text display in DevTools that it might be worth it for us to add an extension method that makes this case easier.

Reply:
Do you mean add a utility function in our devtools code or add something to the tester library?

Reply:
you can add an extension method to our package and it will feel like it is directly part of the tester package in terms of how you call the api.